ohm-js/src/ohm-grammar.ohm

```diff
@@ -109,7 +109,7 @@ Ohm {
    += comment
 
   comment
-    = "//" (~"\n" any)* "\n"  -- singleLine
+    = "//" (~"\n" any)* &"\n"  -- singleLine
```

Comment:
Sorry I'm late to the party!

Should we make the look-ahead expression here `&("\n" | end)` to allow single-line comments to happen at the last line of a file that doesn't have a `"\n"` at the end?

Reply:
Good idea.

Reply:
Done.

Reply:
Cool, thanks @pdubroy!